source/server/admin/config_dump_handler.cc

```diff
@@ -309,7 +309,7 @@ ConfigDumpHandler::dumpEndpointConfigs(const Matchers::StringMatcher& name_match
     float value = cluster.dropOverload().value() * 1000000;
     if (value > 0) {
       auto* drop_overload = policy.add_drop_overloads();
-      drop_overload->set_category("drop_overload");
+      drop_overload->set_category(cluster.dropCategory());
       auto* percent = drop_overload->mutable_drop_percentage();
       percent->set_denominator(envoy::type::v3::FractionalPercent::MILLION);
```

Comment:
Seems that the [drop_percentage](https://github.com/envoyproxy/envoy/blob/b10deb602383972cd71ab0a836c846f7eb91b790/api/envoy/config/endpoint/v3/endpoint.proto#L51) in the input is of type `FractionalPercent` (which is the same as the config-dump.
IMHO it makes more sense to keep the exported value the same (so one can easily compare the config-dump to the input).

Reply:
Is your question that if input is numerator : 20, denominator: HUNDRED, the config dump converted that into MILLION based?

Reply:
Yes. It seems that it overrides the input. I understand that it will also update the numerator, but it just seems unintuitive from a user's perspective. BTW this is not a blocker for this PR, just a comment for a future improvement.

Reply:
BTW, the config dump choose MILLION based, is to cover the case that the drop_ratio is very small, like 0.00001.